components/page/PageTitle.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="bg-[#F1F6FBED] dark:bg-inherit">
+  <div class="bg-grey-200 dark:bg-inherit">
```

Comment:
I'm glad we are removing this arbitrary values. that's quite a door for css disaster